api/v1/articles/index.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @defgroup api_v1_articles Article API requests
+ */
+
+/**
+ * @file api/v1/articles/index.php
```

Comment:
1- how would we determine the path to the file to include? (cf. https://github.com/kaschioudi/pkp-lib/blob/2163-1503-fix/classes/core/APIRouter.inc.php#L85) 
2- this gives opportunity to inject dependencies into the handler if necessary